android/cli/hooks/run.js

```diff
@@ -156,61 +152,61 @@ exports.init = function (logger, config, cli) {
+						builder.target === 'device' && logger.info(`Installing app on device: ${(device.model || device.manufacturer || device.id).cyan}`);
 
 						let intervalTimer = null;
 						const abortTimer = setTimeout(function () {
 							clearTimeout(intervalTimer);
-							logger.error(__('Application failed to install') + '\n');
-							logger.log(__('The current timeout is set to %s ms', String(installTimeout).cyan));
-							logger.log(__('You can increase this timeout by running: %s', (cli.argv.$ + ' config android.appInstallTimeout <timeout ms>').cyan) + '\n');
+							logger.error('Application failed to install\n');
+							logger.log(`The current timeout is set to ${String(installTimeout).cyan} ms`);
+							logger.log(`You can increase this timeout by running: ${(cli.argv.$ + ' config android.appInstallTimeout <timeout ms>').cyan}\n`);
 							if (++failCounter >= deviceInfo.length) {
 								process.exit(1);
 							}
 						}, installTimeout);
 
-						logger.trace(__('Checking if package manager service is started'));
+						logger.trace('Checking if package manager service is started');
 
 						(function installApp() {
 							adb.ps(device.id, function (err, output) {
 								if (err || output.toString().indexOf('system_server') === -1) {
-									logger.trace(__('Package manager not started yet, trying again in %sms...', retryInterval));
+									logger.trace(`Package manager not started yet, trying again in ${retryInterval} ms...`);
 									intervalTimer = setTimeout(installApp, retryInterval);
 									return;
 								}
 
-								logger.trace(__('Package manager has started'));
+								logger.trace('Package manager has started');
 
 								adb.installApp(device.id, builder.apkFile, { logger: logger }, function (err) {
 									if (err) {
 										if (err instanceof Error && (err.message.indexOf('Could not access the Package Manager') !== -1 || err.message.indexOf('Can\'t find service: package') !== -1)) {
-											logger.debug(__('ADB install failed because package manager service is still starting, trying again in %sms...', retryInterval));
+											logger.debug(`ADB install failed because package manager service is still starting, trying again in ${retryInterval} ms...`);
 											intervalTimer = setTimeout(installApp, retryInterval);
 											return;
 										}
 
-										logger.error(__('Failed to install apk on "%s"', device.id));
+										logger.error(`Failed to install apk on "${device.id}"`);
 										err = err.toString();
 										err.split('\n').forEach(logger.error);
 										if (err.indexOf('INSTALL_PARSE_FAILED_NO_CERTIFICATES') !== -1) {
-											logger.error(__('Make sure your keystore is signed with a compatible signature algorithm such as "SHA1withRSA" or "MD5withRSA".'));
+											logger.error('Make sure your keystore is signed with a compatible signature algorithm such as "SHA1withRSA" or "MD5withRSA".');
```

Comment:
I think MD5 is not recommended anymore but I can't find any info about it at the moment, so we can keep it unless I find something else